tests/test_basics.py

```diff
@@ -1,14 +1,16 @@
-import os
-import sys
-relpath = os.path.dirname(__file__) # Find relative path: from current directory to test.py
-abspath = os.path.abspath(".")      # Find absolute path: from / to current directory
-os.chdir(abspath + '/first_timers')
-sys.path.append('./')
-import first_timers
+# import os
```

Comment:
Can we remove these comments.

As we are doing cleanups, I think we should remove these commented code which we are not using.

Reply:
I left them in there, so it is easy for you to revert back in case you changed your mind. 
I will remove them now.

Reply:
@arshadkazmi42 clean up done.